docs/tutorials/create-a-chat-app.mdx

```diff
@@ -276,6 +276,9 @@ peer.sync("https://my.server", true);
 
 This won't work because there is no Earthstar server at `https://my.server`. [Fortunately we have a tutorial for running an Earthstar server](/tutorials/run-a-server).
 
+Beware, that you may need to add an exception in your browser for your local Earthstar server in the cookies and site data section.
+E.g. in Firefox go to "Settings -> Privacy & Security -> Cookies and Site Data -> Manage Exceptions...".
+
```

Comment:
Please let me know if this is the best way and/or place for this comment.

Once this is clarified, I'll take a look on maybe adding a dedicated markdown for highlighting this as a potential error.

Reply:
It's the right place, but maybe? this might be better as a new `<details>` section? *Make sure you browser allows Earthstar to save data** as the heading, or something like that?

Reply:
Good suggestion. I'll take a look on how to change this.

Reply:
I updated the section and moved this part into a `<details>` section.
Let me know, if the updated wording is understandable.